src/Tempest/Support/src/PathHelper.php

```diff
@@ -4,8 +4,67 @@
 
 namespace Tempest\Support;
 
+use Exception;
+use Tempest\Core\Composer;
+use Tempest\Core\Kernel;
+use function Tempest\get;
+
 final readonly class PathHelper
 {
+    public static function root(string ...$paths): string
```

Comment:
This is no longer used, this should be removed

Reply:
Made an issue for it: https://github.com/tempestphp/tempest-framework/issues/615